dev-6/AutomationPractice.Core/log4net.config

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<log4net>
+	<appender name="ConsoleAppender" type="log4net.Appender.ConsoleAppender">
+		<layout type="log4net.Layout.PatternLayout">
+			<conversionPattern value="%date %-5level %logger - %message%newline" />
+		</layout>
+	</appender>
+	<appender name="FileAppender" type="log4net.Appender.FileAppender">
+		<file value = "D:\Logs\MyLogFile.txt"></file>
```

Comment:
This is unsafe way to set path of value. If your tests are running on remote machine where is no disk D you'll get error.